gcloud/monitoring/client.py

```diff
@@ -195,6 +203,129 @@ def metric_descriptor(self, type_,
             display_name=display_name,
         )
 
+    @staticmethod
+    def metric(type_, labels):
+        """Factory for constructing metric objects.
+
+        :class:`~gcloud.monitoring.metric.Metric` objects are typically
+        created to write custom metric values. The type should match the
+        metric type specified in the
+        :class:`~gcloud.monitoring.metric.MetricDescriptor` used
+        create the custom metric::
+
+             >>> metric = client.metric('custom.googleapis.com/my_metric',
+             ...                        labels={
+             ...                            'instance_name': 'my-instance',
+             ...                         })
+
+        :type type_: string
+        :param type_: The metric type name.
+
+        :type labels: dict
+        :param labels: A mapping from label names to values for all labels
+                       enumerated in the associated
+                       :class:`~gcloud.monitoring.metric.MetricDescriptor`.
+
+        :rtype: :class:`~gcloud.monitoring.metric.Metric`
+        :returns: The metric instance.
+        """
+        return Metric(type=type_, labels=labels)
+
+    @staticmethod
+    def resource(type_, labels):
+        """Factory for constructing monitored resource instances.
+
+        Creating a monitored resource (represented by
+        :class:`~gcloud.monitoring.resource.Resource`) object, which is
+        typically used when writing custom
+        :class:`~gcloud.monitoring.timeseries.TimeSeries`.
```

Comment:
Let's make that a proper sentence: "A monitored resource object (a `Resource` instance) is typically used..."